src/main/scala/dev/atedeg/ConfigurationUtils.scala

```diff
@@ -0,0 +1,43 @@
+package dev.atedeg
+
+import better.files.File
+import cats.data.NonEmptyList
+import io.circe.{ Decoder, Json }
+import io.circe.yaml.parser
+import cats.syntax.all._
+
+import Extensions._
+
+object ConfigurationParsing {
+  private val configFile = ".ubidoc.yaml"
```

Comment:
Allow .yml too

Reply:
Ok, I'm fixing this; @vitlinda in the meantime you could add a test that checks that the plugin works with a `.ubidoc.yml` file too